aristanetworks/ansible-avd
# Feat(eos_designs): Extend the functionality of the is_deployed flag to shutdown BGP peerings.

## Change Summary
Having BGP peers in a state other than active, causes monitoring issues, and breaks CVP tests. If hosts are intentionally not deployed, then the BGP peerings should be shutdown.

## Related Issue(s)

Fixes #3604

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Add dict element shutdown: true to neighbors if peer_is_deployed is false.

## How to test
Set is_deployed to false for router and check the bgp configuration on the peers.

## Checklist

### User Checklist

### Repository Checklist
